botbot/Module/MockModule.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace botbot.Module
+{
+    public class MockModule : IMessageModule
+    {
+        public Task<ModuleResponse> Handle(string text, string userId, string channel)
+        {
+            if (text.ToLower().StartsWith("botbot mock"))
+            {
+                string[] splitText = text.Split(" ");
+                string phrase = string.Join(" ", splitText.Skip(2).Take(splitText.Length - 1));
+                Random rand = new Random();
+                string mockText = "";
+                foreach (char c in phrase) {
```

Comment:
I tried to do this with a LINQ `Select` but it was generating the same random number every time